packages/telemetry/src/telemetryReporter.ts

```diff
@@ -181,11 +188,11 @@ export class SpawnedTelemetryReporter extends AsyncCreatable<TelemetryOptions> {
    * Starts the child process, waits, and then stops the child process.
    */
   private beginLifecycle(): void {
-    logTelemetryStatus(this.env, this.logger);
-    if (!isSfdxTelemetryEnabled(this.env)) return;
+    logTelemetryStatus(this.logger, this.config);
+    if (!isSfdxTelemetryEnabled(this.config)) return;
 
     this.start();
-    const insightsTimeout = Number(this.env.getString(SpawnedTelemetryReporter.SFDX_INSIGHTS_TIMEOUT)) || 3000;
+    const insightsTimeout = Number(this.env.getString(TelemetryReporter.SFDX_INSIGHTS_TIMEOUT)) || 3000;
```

Comment:
Seems like we should have an `env.getNumber()`... Then this would be `this.env.getNumber(TelemetryReporter.SFDX_INSIGHTS_TIMEOUT, 3000)` Usually the value after the name is the default value.

Reply:
BTW, this would be a change in kit which isn't in this lerna repo so you don't need to do it in this PR. Mainly just a comment.